src/nvim/tag.c

```diff
@@ -1467,6 +1467,7 @@ find_tags(
 
   help_save = curbuf->b_help;
   orgpat.pat = pat;
+  orgpat.regmatch.regprog = NULL;
```

Comment:
I wonder why Vim doesn't use `{ 0 }` (or memset) to zero-init local structs. Compiler compatiblity or optimization? Bugs from garbage struct values should be avoided.

Reply:
In the `funcexe_T` patches, `memset` 0 was mostly used, but there were a few cases where it wasn't.
I _think_ some later patches changed those to use `memset` too...

So I have no idea. :smile: